lsp-mode.el

```diff
@@ -6361,14 +6360,20 @@ Return a nested alist keyed by symbol names. e.g.
   (when menu-bar-mode
     (lsp--imenu-refresh)))
 
+(defun lsp--check-command-list (command)
+  (cl-assert (seq-every-p (apply-partially #'stringp) command) nil
+             "Invalid command list"))
+
 (defun lsp-resolve-final-function (command)
   "Resolve final function COMMAND."
   (-let [command (if (functionp command) (funcall command) command)]
     (cl-etypecase command
       (list
-       (cl-assert (seq-every-p (apply-partially #'stringp) command) nil
-                  "Invalid command list")
+       (lsp--check-command-list command)
        command)
+      (vector
```

Comment:
Is there case of command is vector?

Reply:
Not really, I just thought that it might be a call feature.